server/config.go

```diff
@@ -26,6 +26,12 @@ type Config struct {
 	// Strategy to use for selecting the vertices/edges to inspect.
 	// Random if True; first N otherwise
 	SchemaRandomSample bool
+	// Configure how the server logs requests
+	RequestLogging struct {
+		Disable bool
```

Comment:
Maybe switch this to Enable, so the default value (false) means no request logging takes place.

Reply:
Will do.